history/characters/440000_sayaadi.txt

```diff
@@ -0,0 +1,45 @@
+#Lowborn
+440000={ 
+	name=Moora
+	female=yes
+	dna="bgtstrhdras"
+	culture=sayaadi religion=burning_legion_religion
+	martial=6 diplomacy=5 stewardship=6 intrigue=6 learning=8
+	add_trait=flamboyant_schemer
+	add_trait=class_shadow_priest_7
+	add_trait=fair
+	add_trait=lustful
+	1.4.17={ birth=yes add_trait=creature_sayaadi immortal_age=18 add_trait=being_demon }
+	2000.6.21={ death=yes }
+}
+440001={ 
+	name=Salia
+	female=yes
+	dna="fgthcsdvthj"
+	culture=sayaadi religion=burning_legion_religion
+	martial=4 diplomacy=6 stewardship=6 intrigue=7 learning=7
+	add_trait=flamboyant_schemer
+	add_trait=class_warlock_7
+	add_trait=fair
+	add_trait=lustful
+	1.4.17={ birth=yes add_trait=creature_sayaadi immortal_age=18 add_trait=being_demon }
+	2000.6.21={ death=yes }
+}
+
+#dynasty=147000
+#Demonic rulers of Hyjal
+440002={ 
```

Comment:
Don't look sayaadi enough to me, tbh

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/76706994-c5876d00-66fc-11ea-9b86-7eb635b0fb4a.png)

</details>

Reply:
Shaved them in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/401/commits/ea054f428315c4053222488744792a73f3bdae36. Should look better.